docs/content/en/docs/packages/packagecontroller.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Managing the package controller"
+linkTitle: "Package controller"
+weight: 4
+aliases:
+    /docs/tasks/packages/packagecontroller/
+---
+
+### Installing the package controller
+
+{{% alert title="Important" color="warning" %}}
+The package controller installation creates a package bundle controller resource for each cluster, thus allowing each to activate different package bundle version. Ideally, you should never delete this resource because it would mean losing that information and upon re-installing, the latest bundle will be selected. However, you can always go back to the previous bundle version. For more information, see [Managing package bundles.]({{< relref "./packagebundles" >}})
+{{% /alert %}}
+
+The package controller is typically installed during cluster creation, but may be disabled intentionally in your `cluster.yaml` by setting `spec.packages.disable` to `true`.
+
+If you created a cluster without the package controller or if the package controller was not properly configured, you may need to manually install it.
+
+1. Enable the package controller in your `cluster.yaml`, if it was previously disabled:
+    ```yaml
+    apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+    kind: Cluster
+    metadata:
+      name: mgmt
+    spec:
+      packages:
+        disable: false
+    ```
+
+2. Make sure you are authenticated with the AWS CLI. Use the credentials you set up for packages. These credentials should have [limited capabilities]({{< relref "./prereq#setup-authentication-to-use-curated-packages" >}}):
+
+    ```bash
+    export AWS_ACCESS_KEY_ID="your*access*id"
+    export AWS_SECRET_ACCESS_KEY="your*secret*key"
+    export EKSA_AWS_ACCESS_KEY_ID="your*access*id"
+    export EKSA_AWS_SECRET_ACCESS_KEY="your*secret*key"
+    ```
+
+3. Verify your credentials are working:
+    ```shell
+    aws sts get-caller-identity
+    ```
+
+4. Authenticate docker to the private AWS ECR registry on account `783794618700` with your AWS credentials. It houses the EKS Anywhere packages artifacts and authentication is required to pull imagaes from it.
+    ```bash
+    aws ecr get-login-password | docker login --username AWS --password-stdin 783794618700.dkr.ecr.us-west-2.amazonaws.com
+    ```
+
+5. Verify you can pull an image from the packages registry:
+    ```bash
+    docker pull 783794618700.dkr.ecr.us-west-2.amazonaws.com/emissary-ingress/emissary:v3.5.1-bf70150bcdfe3a5383ec8ad9cd7eea801a0cb074
+    ```
+    If the image downloads successfully, it worked!
+
+6. Now, install the package controller using the eks anywhere package CLI:
```

Comment:
```suggestion
6. Now, install the package controller using the eks anywhere package CLI:
```
Is the package literally named "eks anywhere". If not change that to "EKS Anywhere".

Reply:
Yes, EKS Anywhere makes sense here. It’s referring to the EKS Anywhere Packages CLI